### ISSUE TITLE
Keyboard navigation of the Calendar does not work with Screen readers

### ISSUE THREAD

**Issue by vveesseelliinnaa** [telerik/kendo-react#1588]
Ticket ID: 1610493

The keyboard navigation does not function correctly when the Calendar is used with JAWS (or NVDA) the keyboard navigation.

Steps to reproeduce:
1. Run the screen reader and open https://stackblitz.com/edit/react-8gmkw3?file=app%2Fmain.jsx
2. Open the calendar popup
3. Press the ArrowLeft key (or alt + arrow)
4. Press Enter

Result: with NVDA, it picks the first of the month (sometimes it goes back in years) and with JAWS nothing happens

**Comment by iadnonov** [comment]
Calendar valuate axe error: `Scrollable region must have keyboard access`
Can be tested in the following page: https://www.telerik.com/kendo-react-ui/components/dateinputs/calendar/keyboard-navigation/

Ticket ID: 1640852

**Comment by iadnonov** [comment]
Reported again ticket ID: 1655953
